pkg/bmtpool/bmtpool.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package bmtpool
+
+import (
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/ethersphere/bmt/pool"
+)
+
+var instance pool.Pooler
+
+func init() {
+	instance = pool.New(8, swarm.BmtBranches)
+}
+
+func Pool() pool.Pooler {
```

Comment:
How about to add Get at Put functions in this package and instead calling `bmtpool.Pool().Get()`, just to have `bmtpool.Get()`? And to remove the Pool() function?